docs/site/Binding.md

```diff
@@ -487,6 +491,49 @@ const binding = ctx.bind('my-key').toClass(MyService);
 
 " %}
 
+#### When to call createBindingFromClass
+
+If you are **not** working with a LoopBack artifact, then calling
```

Comment:
This is misleading. All things are LoopBack artifacts. We should say something like:

```
To manually register classes or provider classes that have extra metadata added by the `@bind` decorator, 

Please note that if such classes are placed in directories such as `src/controllers` and 'src/services' will be automatically registered by [the boot process](Booting-application.md).
```